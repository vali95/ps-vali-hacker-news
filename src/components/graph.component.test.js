import React from 'react';

import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import Graph from './graph.component'

configure({adapter:new Adapter()})

describe("Graph Component Renders Correctly. Snapshot test",()=>{
   
    it("Should Recive Data in Props",()=>{         
        const wrapper = shallow(<Graph/>)        
        expect(wrapper).toMatchSnapshot();
    })
})