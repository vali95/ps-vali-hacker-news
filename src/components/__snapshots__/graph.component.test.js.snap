// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graph Component Renders Correctly. Snapshot test Should Recive Data in Props 1`] = `ShallowWrapper {}`;
